packages/react-components/style/output-components-style.css

```diff
@@ -93,6 +93,14 @@ canvas {
     align-self: center;
 }
 
+.analysis-failed-message {
+    font-size: 20px;
+    margin-top: -20px; /* So text is vertically aligned relative to its center rather than its top */
+    transform: translate(0, 50%);
+    margin-left: auto;
+    margin-right: auto;
+}
```

Comment:
CSS recommendations welcome! I have no knowledge of best practices when it comes to CSS.